Translations/Portugues Brasil.csv

```diff
@@ -3909,4 +3909,20 @@ research.scanning.desc,Adiciona a ferramenta para localizar recursos dentro do S
 system.scan.basic.unlocked,O recurso de Rastreamento foi adicionado ao Mapa Estelar. É necessário ter no mínimo um <b>Radar</b> para rastrear recurso dentro do Sistema.,,provisório
 system.scan.advanced.unlocked,Recurso de Rastreamento expandido. Construa <b>Rastreadores</b> para aumentar seu alcance.,,provisório
 ship.computer.cap.unlock,O limite de Computadores Ativos foi aumentado para <b>{0}</b> em uma rede.,,
-upgrade.unlocked,Overclock disponível,,provisório
+upgrade.unlocked,Melhoria disponível,,provisório
+obj.materials.caoutchouc,Resina,,
+obj.materials.caoutchouc.desc,O famoso liquido branco extraído da Seringueira.,,
+obj.plants.rubbertree,Seringueira,,
+obj.plants.rubbertree.desc,Uma alta arvore que produz uma resina muito cobiçada.,,
+objects.furniture.pantry,Guarda comida,,need test
+objects.furniture.pantry.desc,"Local usado para armazenar mantimentos.",,
+objects.devices.pentagramstatue,Estatua especial,,wip
+objects.devices.pentagramstatue.desc,Uma estatua com propriedades desconhecidas.,,wip
+objects.furniture.largepantry,Dispensa,,need test
+objects.furniture.largepantry.desc,"Uma versão muito maior do Guarda comida , podendo armazenas mantimentos como grãos de cafe, farinha, grãos , etc.",,

```

Comment:
Would be nice to have some consistency with spaces around commas. The correct way is to have no space before comma, and one space after it.

Reply:
okay, i was called at the time so i forgot, will fix for next.